RoadMap_VK/RoadMap_VK/Model/News.swift

```diff
@@ -0,0 +1,10 @@
+// News.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// News
```

Comment:
```suggestion
/// News
```
Коменты либо все на русском либо все на английском перемешивать нельзя